tfhe/src/integer/server_key/radix_parallel/scalar_sub.rs

```diff
@@ -112,4 +116,57 @@ impl ServerKey {
             self.full_propagate_parallelized(ct);
         }
     }
+
+    pub fn unsigned_overflowing_scalar_sub_parallelized<T>(
```

Comment:
any reason you don't do the sale as for add creating the sub_assign function and calling it in the non assign version ?